src/status_im/ui/components/desktop/events.cljs

```diff
@@ -0,0 +1,16 @@
+(ns status-im.ui.components.desktop.events
+  (:require [status-im.utils.handlers :as handlers]
+            [status-im.utils.handlers-macro :as handlers-macro]))
+
+(defn set-render-user [user {:keys [db]}]
+  "sets the user which should be visible on the desktop tab: profile"
+  (when user
+    {:db (assoc db :desktop/render-user user)}))
+
+(handlers/register-handler-fx
+ :show-desktop-tab
```

Comment:
How does the logic exactly works?

Why can't we use `current-account` directly instead of storing it in the database under a different key (`desktop/render-user`)?

Reply:
The goal was to be able to re-use the same tab for displaying either the current user's information or the profile of the user the "current user" is chatting with. 

Previously the user profile tab subscribed to the current account, and I introduced this new key so that it could be more flexible. `desktop/render-user` can take on a value of either the current account or the person who is being chatted with. The different values are set based on whether the app user clicks on their own profile or someone else's. 

Ideally it would be nice to pass the account as an argument to the tab profile component, but given how it is currently structured, that struck me as a larger refactor beyond the scope of this change. Happy to make that as part of this or either another PR if that seems more sensible in the long run.

Reply:
@rcullito I see, makes sense, I would recommend to move away from denormalising and do as you proposed, as otherwise the view will not re-render if the subscription is cached and a contact update is received, unless you also update the rendered-user, which is probably something we want to avoid, and try to keep state normalized unless necessary.

This PR another PR is all fine by me :)

Reply:
@cammellos Thanks for the review! Right, good point about the benefits of re-rendering. And sounds good, will aim to "re-normalize" in this PR then before it proceeds too much further. :+1: